oceanprefilter/prefilter_test.go

```diff
@@ -0,0 +1,119 @@
+package oceanprefilter
+
+import (
+	"fmt"
+	"image"
+	"math"
+	"os"
+	"path/filepath"
+	"testing"
+	"image/jpeg"
+	xgb "github.com/Elvenson/xgboost-go"
+	"github.com/Elvenson/xgboost-go/activation"
+)
+
+
+func TestGeneralModel(t * testing.T) {
+
+	f, err := os.Open("numbered_data/1000.jpg")
+	fmt.Print(err)
+	defer f.Close()
+	img, _, err := image.Decode(f)
+
+	rc := runConfig{}
+	ensemble, err := xgb.LoadXGBoostFromJSONBytes(modelbytes,
+		"", 2, 8, &activation.Softmax{}) // wrong activation logistic is 1 class only
+	if err != nil {
+		panic(err)
+	}
+	rc.model = ensemble
+	rc.threshold = 0.25
+	rect := image.Rectangle{
+		Min: image.Point{X: 250, Y: 350},
+		Max: image.Point{X: 580, Y: 480},
+	}
+	rc.excludedZone = &rect
+	
+	res, err := make_inference(img, rc)
+	if err != nil {
+		fmt.Printf("err")
+	}
+
+	fmt.Printf("output result is %+v\n", res)
```

Comment:
Do not use print statements for tests, and instead check that the result value is what is expected, and that the error is nil